source/Classroom/Basics/Account/adding_an_additional_dedicated_ip_to_your_account.md

```diff
@@ -18,4 +18,4 @@ We do not recommend adding a new IP to automated warmup if you have no other IP
 
```

Comment:
On line 13, where it says "click the "Add a Dedicated IP" button

Change to:
Click **Add a Dedicated IP**.